content/en/docs/tasks/administer-cluster/highly-available-master.md

```diff
@@ -124,7 +124,7 @@ This operation may be sped up by migrating etcd data directory, as described [he
 
 ## Implementation notes
 
-![ha-master-gce](/images/docs/ha-master-gce.png)
```

Comment:
This looks more like a tooling issue, I feel like the original relative address seems better.